runtime/fundamentals/testing.md

```diff
@@ -0,0 +1,757 @@
+---
+title: "Testing"
+oldUrl:
+  - /runtime/manual/advanced/language_server/testing_api/
+  - /runtime/manual/basics/testing/
+  - /runtime/manual/basics/testing/coverage/
+  - /runtime/manual/basics/testing/assertions/
+  - /runtime/manual/basics/testing/mocking/
+  - /runtime/manual/basics/testing/behavior_driven_development
+  - /runtime/manual/testing/documentation/
+  - /runtime/manual/basics/testing/sanitizers/
+  - /runtime/manual/basics/testing/snapshot_testing/
+---
+
+Deno provides a built-in test runner for writing and running tests in both
+JavaScript and TypeScript. This makes it easy to ensure your code is reliable
+and functions as expected without needing to install any additional dependencies
+or tools.
+
+The Deno test runner fully supports typescript and async tests and allows for
+fine-grained control over permissions for each test, enhancing your code
+security.
```

Comment:
We already mention that the test runner supports TS in the paragraph just before this one. I'm not sure if we explicitly need to mention "async tests" as a feature. I don't know of a test runner which doesn't have this. I like the part about passing individual permissions to tests though, we should keep that. Can't come up with a good phrasing right now.